src/api/submission/utils.py

```diff
@@ -8,7 +8,17 @@
 from rest_framework.exceptions import ValidationError
 
 from ..mocks import MockRequest
-from ..models import AuditRecord, CollectRecord
+from ..models import (
+    BENTHICLIT_PROTOCOL,
+    BENTHICPIT_PROTOCOL,
+    FISHBELT_PROTOCOL,
+    HABITATCOMPLEXITY_PROTOCOL,
+    BLEACHINGQC_PROTOCOL,
+    BENTHIC_PHOTO_QUADRAT_TRANSECT,
```

Comment:
Don't we need to add the new BQT protocol to the `elif` blocks below?

Reply:
We will but that'll be another ticket for validation/submission